tools/userland-incorporator

```diff
@@ -111,7 +111,10 @@ def get_incorporations(repository, publisher, inc_version='5.11'):
     for package in packages:
         pkg_name = package['name']
         pkg_version = package['version']
-        consolidations = package['org.opensolaris.consolidation'][0]['value']
+        try:
+            consolidations = package['org.opensolaris.consolidation'][0]['value']
+        except:
+            consolidations = []
```

Comment:
Any idea what package(s) needs this?  Thanks.

Reply:
Unfortunately no. And so far printf debugging can only work on the prod server. So ideas how to log this are welcome in later changes.